docs/dev/testing.rst

```diff
@@ -261,3 +262,45 @@ database containing all years of FERC 1 data (equivalent to ``tox -e ferc1_schem
 .. code-block:: console
 
   $ pytest --etl-settings test/settings/full-integration-test.yml test/integration/etl_test.py::test_ferc1_schema
+
+-------------------------------------------------------------------------------
+Data Validation
+-------------------------------------------------------------------------------
+Once the PUDL ETL pipeline has processed all of the available data, we have a
+collection of tests that can be run to verify a collection of expectations
+about what the outputs should look like. These data validation tests are
```

Comment:
Maybe include an example of one of these expectations? Might be good to say that we aren't actually validating every row of data, rather, we are validating that we correctly read the raw data.

Reply:
Not sure what you're getting at here exactly. We're checking more than that the raw input data has been read -- we're looking at the distribution of values in a bunch of columns (coal heat content, per ton etc.) to make sure that they aren't outside of reasonable bounds, since we've transformed those values, fixed some outliers, etc.

Reply:
I think I meant including a reference to the two "types" of data validation stated below: 

```
The data validation cases that pertain to the contents of the data tables are currently stored as part of the pudl.validate module.

The expected number of records in each output table is stored in the validation test modules under test/validate as pytest parameterizations.
```